src/routers/xcm/Router.sol

```diff
@@ -174,9 +174,23 @@ contract ConnectorXCMRouter {
     function message_length_scale_encoded(bytes memory message) internal pure returns (bytes memory) {
         bytes29 _msg = message.ref(0);
 
-        if (ConnectorMessages.isTransferTrancheTokens(_msg)) {
+        if (ConnectorMessages.isTransfer(_msg)) {
+            return hex"8501";
```

Comment:
The function-level comment explains how these scale-encoded length values are obtained.